test-lit/lit_port.py

```diff
@@ -0,0 +1,256 @@
+""" lit_port.py
```

Comment:
I imagine that as we want to merge with LLVM. This header needs to be changed to:
# -*- Python -*-
I believe you can remove the licence part. I don't see that in the python files inside LLVM. I can be wrong, so double check.

Reply:
@kiranchandramohan asked me to add the license I think, and I'm not 100% sure, but I remember Rich confirming this. I will double check though.

Reply:
I don't remember asking you to add license. But i can be forgetful. @RichBarton-Arm probably asked.
I don't see the license in lit.cfg.py and other python files in the test directory but i see the license in several other python files. See the grep below.
~/test_build/organic-clang$ grep Apache `find -name *.py` | wc -l
62